Header header
vision_msgs/BoundingBox2D[] bounding_boxes
